plugin/template.py

```diff
@@ -41,6 +42,10 @@ def _plugin_asset_path(asset_path: str) -> str:
     return f"Packages/{PACKAGE_NAME}/plugin/assets/{asset_path}"
 
 
+def sanitize_path(path: str) -> str:
+    return unquote(path.replace("file://", ""))
```

Comment:
is this just https://github.com/sublimelsp/LSP/blob/fe795a145a0d7b666caf00d15abe6b22380168e9/plugin/core/url.py#L46 ?